object_real: false
iiwa_real: false
manual_mode: true
hollow: false


trials:
  center: [0.55, 0.3, 0.165]
  range: [0.3, 0.4]
  N: 12 #192


ee_offset:
  h: 0.4                # horizontal offset for the end effector behind the object's center before moving to hit
  v: 0.25             # vertical offset


table:
  properties:           # keep in mind that the iiwa base has a max radius of 0.3
    size:
      x: 0.8
      y: 2.0
      z: 0.05
    COM:
      x: 0.0
      y: 0.0
      z: 0.0
    mass: 100
    mu: 0.5
    mu2: 0.5
  initial_pos:
    x: 0.60
    y: 0.5
    z: 0.025

box:
  properties:
    size:
      x: 0.26
      y: 0.26
      z: 0.23
    COM:            #relative to geometric center
      x: 0.0
      y: 0.0
      z: 0.0
    mass: 0.363
    mu: 0.1
    mu2: 0.1
  initial_pos:
    x: 0.4
    y: 0.1
    z: 0.165
  
mini:
  properties:
    size:
      x: 0.1
      y: 0.1
      z: 0.1
    COM:            #relative to geometric center
      x: 0.0
      y: 0.0
      z: 0.0
    mass: 0.3
    mu: 0.03
    mu2: 0.03
  initial_pos:      #relative to big box center
    x: 0.0
    y: -0.065
    z: -0.065



#hit:
#  speed: 1.0
#  direction: 0.0
